### PR TITLE
Add Feedback Fish

### PR DESCRIPTION
**Pull request by adoniasvitorio**
Hi @zenorocha !

My new suggest is **Feedback Fish** as product.

Thanks !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@
 | [Budibase](https://budibase.com)       | Budibase helps you build internal tools on your own infrastructure in minutes.                              | 2019         |
 | [Chromatic](https://www.chromatic.com) | Chromatic automates gathering UI feedback, visual testing, and documentation, so developers can iterate faster with less manual work.                             | 2017         |
 | [Doppler](https://doppler.com)       | Doppler helps developers manage their API keys across their projects.                              | 2018         |
+| [Feedback Fish](https://feedback.fish/)                  | A simple way to collect feedback & make your customers happy..                                          | 2020         |
```
Comment (zenorocha) [inline README.md:74]: Double `..`

**Review by zenorocha — CHANGES_REQUESTED** [review]
Looking good! Once you fix the typo, I can merge the PR ;)

**Comment by adoniasvitorio** [comment]
fix `..`

**Comment by bukinoshita** [comment]
Looks good to me! Let's merge 🎉

**Pull request merged**